src/index.ts

```diff
@@ -3,15 +3,27 @@ import express from 'express';
 import compression from 'compression';
 import cors from 'cors';
 import api from './api';
+import webhook from './webhook';
 import log from './helpers/log';
 import './lib/queue';
+import { name, version } from '../package.json';
 
 const app = express();
 const PORT = process.env.PORT || 3000;
 
 app.use(express.json({ limit: '4mb' }));
 app.use(cors({ maxAge: 86400 }));
 app.use(compression());
-app.use('/', api);
+app.use('/api', api);
+app.use('/', webhook);
```

Comment:
I think it should be `/webhook` here and `/` in webhook.ts just to avoid confusion with `/` route below :)

Reply:
then the `log.info(`[http] POST /webhook`);` inside webhook will not match the router ...mhhh

Reply:
You think we can get more webhook routes in the future? if not we can just make in `app.post` here, no need of a separate router, if we think we may get more webhooks - then we should have a route like `webhook/events` so we can recognize them easily

Reply:
file will grow for sure, at least 2 other PR are using it. But we will have only one single /webhook endpoint for all webhooks